app/models/case.js

```diff
@@ -23,7 +23,7 @@ export default Model.extend({
     return A(this.get('pieces').toArray()).sort((pieceA, pieceB) => compareFunction(new VRDocumentName(pieceA.get('name')), new VRDocumentName(pieceB.get('name'))));
   }),
 
-  latestSubcase: computed('subcases.@each', function() {
+  latestSubcase: computed('subcases.[]', 'id', 'store', function() {
```

Comment:
computen op `store` ?

Reply:
`store` is in praktijk idd niet iets dat gaat veranderen. Ik vermoed dat het is omdat men niet meteen `store` in een `computed` verwacht, dat er ook geen uitzondering voor in de linter zit. Van mij mag hij er ook uit met een exception toegevoegd ... eender.